dags/springer/repository.py

```diff
@@ -0,0 +1,55 @@
+import os
+from typing import IO
+
+import boto3
+from botocore.client import Config
+from common.repository import IRepository
+from mypy_boto3_s3.service_resource import _Bucket
+
+
+class SpringerRepository(IRepository):
+    ZIPED_DIR: str = "raw/"
+    EXTRACTED_DIR: str = "extracted/"
+    s3: _Bucket = None
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.s3 = boto3.resource(
+            "s3",
+            endpoint_url=os.getenv("SPRINGER_REPO_SOURCE", "http://localhost:9000"),
+            aws_access_key_id=os.getenv("SPRINGER_REPO_USERNAME", "airflow"),
+            aws_secret_access_key=os.getenv("SPRINGER_REPO_PASSWORD", "Airflow01"),
+            config=Config(signature_version="s3v4"),
+            region_name=os.getenv("SPRINGER_REPO_REGION", "us-east-1"),
+        ).Bucket(os.getenv("SPRINGER_REPO_BUCKET", "springer"))
```

Comment:
Sorry if I wasn't clear, I meant to have a service for s3 that we initialize with the same config and just configure the bucket per publisher.

Reply:
Ohh got you, will do

Reply:
Done